File: feinblelib/src/main/java/com/bbg/feinblelib/callback/BleWriteCallback.java
package com.bbg.feinblelib.callback;

import com.bbg.feinblelib.exception.BleException;

public abstract class BleWriteCallback extends BleBaseCallback{

    public abstract void onWriteSuccess(int current, int total, byte[] justWrite);

    public abstract void onWriteFailure(BleException exception);

}

File: feinblelib/src/main/java/com/bbg/feinblelib/callback/BleWriteCallback.kt
package com.bbg.feinblelib.callback

import com.bbg.feinblelib.exception.BleException

abstract class BleWriteCallback : BleBaseCallback() {
    abstract fun onWriteSuccess(current: Int, total: Int, justWrite: ByteArray?)
    abstract fun onWriteFailure(exception: BleException?)
}